docs/examples/airbnb/index.html

```diff
@@ -21,37 +21,72 @@
   <![endif]-->
 </head>
 <body>
-  <div class="container-fluid" id="main">
-    <header>
+
+  <!-- Header -->
```

Comment:
I get the comment sections but most of the time we have
```html
<!-- Header -->
<header>
...
</header>
<!-- /Header -->
```

This look redundant and noise to me always